alert.html

```diff
@@ -0,0 +1,43 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../vui-colors/colors.html">
+
```

Comment:
So before we proceed, we should nail down (with design's help) all the terminology for the types (alert vs. warning vs. info vs. ??) and modes (inline, toast, etc.). That will drive what we call and how we build these web components.